projects/tarantool/build.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
+# Copyright 2021 Sergey Bronnikov <sergeyb@tarantool.org>
```

Comment:
Can you remove this line please?

Reply:
Sure, removed.

BTW, did you know that a number of projects have additional copyrights?
Just run `grep -R Copyright projects/ | grep -v Google` in a root directory.

Reply:
Thanks.
I didn't know this. Thanks for pointing it out.